adlfs/multithread.py

```diff
@@ -183,6 +185,7 @@ def get_chunk(adlfs, src, dst, offset, size, blocksize, retries=MAXRETRIES,
                             logger.debug('Aborting %s, byte offset %s', dst, o)
                             raise
```

Comment:
I now return the number of bytes written and the exception as a string (None if not present).